dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -1114,6 +1095,7 @@ void DAGQueryBlockInterpreter::executeImpl(DAGPipeline & pipeline)
     else if (query_block.source->tp() == tipb::ExecType::TypeWindow)
     {
         handleWindow(pipeline, query_block.source->window());
+        restorePipelineConcurrency(pipeline);
```

Comment:
move 1098 to L1100.
See https://github.com/pingcap/tiflash/issues/3804.

Reply:
> move 1098 to L1100. See #3804.

done